packages/preview-middleware-client/test/unit/adp/quick-actions/fe-v2.test.ts

```diff
@@ -615,69 +672,92 @@ describe('FE V2 quick actions', () => {
+                                        enabled: true,
+                                        label: `'MyTable' table`
+                                    }
+                                ],
+                                enabled: true,
+                                id: 'listReport0-create-table-custom-column',
+                                kind: 'nested',
+                                title: 'Add Custom Table Column'
+                            }
+                        ]
+                    }
+                ];
 
                 expect(sendActionMock).toHaveBeenCalledWith(
-                    quickActionListChanged([
-                        {
-                            title: 'LIST REPORT',
-                            actions: [
-                                {
-                                    'kind': 'nested',
-                                    id: 'listReport0-create-table-action',
-                                    title: 'Add Custom Table Action',
-                                    enabled: true,
-                                    children: [
-                                        {
-                                            children: [],
-                                            enabled: true,
-                                            label: `'MyTable' table`
-                                        }
-                                    ]
-                                },
-                                {
-                                    'children': [
-                                        {
-                                            'children': [],
-                                            enabled: true,
-                                            'label': `'MyTable' table`
-                                        }
-                                    ],
-                                    'enabled': true,
-                                    'id': 'listReport0-create-table-custom-column',
-                                    'kind': 'nested',
-                                    'title': 'Add Custom Table Column'
-                                }
-                            ]
-                        }
-                    ])
+                    quickActionListChanged(expectedActions(testCase.showTooltip as boolean) as any)
                 );
-
                 await subscribeMock.mock.calls[0][0](
                     executeQuickAction({ id: 'listReport0-create-table-action', kind: 'nested', path: '0' })
                 );
-
-                expect(DialogFactory.createDialog).toHaveBeenCalledWith(
-                    mockOverlay,
-                    rtaMock,
-                    'AddFragment',
-                    undefined,
-                    {
-                        aggregation: 'content',
-                        title: 'QUICK_ACTION_ADD_CUSTOM_TABLE_ACTION'
-                    }
-                );
+                if (!testCase.showTooltip) {
```

Comment:
This is wrong way to check expectation.
With this if condition you are just bypassing the call check and reducing test reliability.
You have to do it like expect(DialogFactory.createDialog).toHaveBeenCalledTimes(testCase.showTooltip ? 0 : 1) followed by more detailed call arguments check

showTooltip option also is not explaining test case.
Better to name it like isTableWithoutToolbar?: boolean